layout_and_module_task/resources/views/admin/tasks/edit.blade.php

```diff
@@ -1,88 +1,90 @@
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Type</label>
-            <input type="text" class="form-control" id="exampleFormControlInput1" name="type"
-                value="{{ $list['type'] }}" />
-            @error('type')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Status</label>
-            <input type="text" class="form-control" id="exampleFormControlInput1" name="status"
-                value="{{ $list['status'] }}" />
-            @error('status')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Start_date</label>
-            <input type="date" class="form-control" id="exampleFormControlInput1" name="start_date"
-                value="{{ $list['start_date'] }}" />
-            @error('start_date')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Due_date</label>
-            <input type="date" class="form-control" id="exampleFormControlInput1" name="due_date"
-                value="{{ $list['due_date'] }}" />
-            @error('due_date')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Assignee</label>
-            <input type="text" class="form-control" id="exampleFormControlInput1" name="assignee"
-                value="{{ $list['assignee'] }}" />
-            @error('assignee')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Estimate</label>
-            <input type="text" class="form-control" id="exampleFormControlInput1" name="estimate"
-                value="{{ $list['estimate'] }}" />
-            @error('estimate')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <div class="mb-3">
-            <label for="exampleFormControlInput1" class="form-label">Actual</label>
-            <input type="text" class="form-control" id="exampleFormControlInput1" name="actual"
-                value="{{ $list['actual'] }}" />
-            @error('actual')
-                <div class="alert alert-danger">{{ $message }}</div>
-            @enderror
-        </div>
-        <input type="submit" class="btn btn-success" style="margin: 5% 50%; width: 10%" value="Update" />
-    </form>
+    @foreach ($tasks as $task)
```

Comment:
vì ở trên em lấy ra mảng nên chỗ này màn hình edit mà lại phải for nè, check lại giúp anh nha, tương tự cho những màn hình khác